### PR TITLE
Add configurable cornerRadius

### PR DESCRIPTION
**Pull request by Marcocanc**
###  Summary

This PR adds support for letting any `PanModalPresentable` set the desired corner radius on the modal.
It also simplifies how the mask path is created.

### Requirements

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackhq/PanModal/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

* [ ] I've written tests to cover the new code and functionality included in this PR.

### CODE REVIEW

File: PanModal/Controller/PanModalPresentationController.swift [hunk 1]
```diff
@@ -797,31 +796,17 @@ private extension PanModalPresentationController {
...
-
-        let path = UIBezierPath()
-        path.move(to: CGPoint(x: 0, y: Constants.cornerRadius))
-
-        // 1. Draw left rounded corner
-        path.addArc(withCenter: CGPoint(x: path.currentPoint.x + Constants.cornerRadius, y: path.currentPoint.y),
-                    radius: Constants.cornerRadius, startAngle: .pi, endAngle: 3.0 * .pi/2.0, clockwise: true)
+        let radius = presentable?.cornerRadius ?? 0
+        let path = UIBezierPath(roundedRect: view.bounds,
+                                byRoundingCorners:[.topRight, .topLeft],
```
Comment (tun57) [inline PanModal/Controller/PanModalPresentationController.swift]: nit: space between `:` & `[.top..`

File: PanModal/Controller/PanModalPresentationController.swift [hunk 2]
```diff
@@ -797,31 +796,17 @@ private extension PanModalPresentationController {
...
+                                cornerRadii: CGSize(width: radius, height: radius))
 
         // 2. Draw around the drag indicator view, if displayed
```
Comment (tun57) [inline PanModal/Controller/PanModalPresentationController.swift]: nit: can we remove the numbered points now that we've cut down the complexity of this func 😄 i.e. `2.`, `6.` & `7.`

File: PanModal/Presentable/PanModalPresentable+Defaults.swift [hunk 3]
```diff
@@ -77,6 +77,10 @@ public extension PanModalPresentable where Self: UIViewController {
         return isPanModalPresented
     }
 
+    var cornerRadius: CGFloat {
```
Comment (tun57) [inline PanModal/Presentable/PanModalPresentable+Defaults.swift]: Can you add this to `testPresentableDefaults` in `PanModalTests` as well

**Comment by TosinAF** [comment]
To make things simple. I'm going to merge & then fix the PR feedback. Happening in a few..

**Comment by TosinAF** [comment]
Thanks so much for your contributions! I was going to make a cocoapod release on Monday. Let me know if you need this changes on cocoapods sooner.

**Comment by Marcocanc** [comment]
Happy to contribute! I use Carthage, so I'll be fine :)

**Pull request merged**